components/accelerator/nvidia/gsp-firmware-mode/config.go

```diff
@@ -9,6 +9,11 @@ import (
 
 type Config struct {
 	Query query_config.Config `json:"query"`
+
+	NvidiaSMICommand         string `json:"nvidia_smi_command"`
+	NvidiaSMIQueryCommand    string `json:"nvidia_smi_query_command"`
+	IbstatCommand            string `json:"ibstat_command"`
+	InfinibandClassDirectory string `json:"infiniband_class_directory"`
```

Comment:
we are passing these everywhere. can we use an embedded struct

Reply:
@ccding Addressed, PTAL. Thanks.